testdata/repos/eclass/eclass/reserved.eclass

```diff
@@ -0,0 +1,21 @@
+# @ECLASS: reserved.eclass
+# @MAINTAINER:
+# Random Dev <random.dev@gentoo.org>
```

Comment:
Let's use example.com.

Reply:
Bikeshed alarm. :smile:

How about this?
```suggestion
# Larry the Cow <larry@example.org>
```

Reply:
I need to remember Larry - we need to increase his presence in pkgcore!

Reply:
WFM!